cognite/neat/rules/models/dms/_rules.py

```diff
@@ -322,15 +323,6 @@ def matching_version_and_space(cls, value: SheetList[DMSView], info: ValidationI
             warnings.warn(issues.dms.ViewModelSpaceNotMatchingWarning(different_space, metadata.space), stacklevel=2)
         return value
 
-    @field_validator("views")
```

Comment:
Why is this removed?

Reply:
Bad merged, it is included in the validation above.